yklstartdeepdoc.py

```
@@ -0,0 +1,1095 @@
+                        stats[layout_type] += 1
+                    else:
+                        stats["other"] += 1
+                    logger.debug(f"  提取图片: {img_name}")
+                except Exception as e:
+                    logger.warning(f"  提取图片失败 [{layout_type}] #{idx}: {e}")
+            else:
+                # 非图表类型（text/title 等），丢弃 PIL Image 避免后续重复保存
+                item["image"] = None
+
+        if saved:
+            logger.info(f"\n图片提取统计:")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)